### PR TITLE
Add Float UI

### PR DESCRIPTION
**Pull request by JuanPabloDiaz**
## Fixes Issue

#2268

## Changes proposed

- Add Float UI 

- Fix subcategory for Material Tailwind

### CODE REVIEW

File: database/frontend/ui-libraries.json [hunk 1]
```diff
@@ -1,65 +1,72 @@
...
+    "name": "Seamless UI",
+    "url": "https://seamlessui.vercel.app/",
+    "description": "The next gen UI made with simple HTML and Tailwind CSS. It's compatible with simple HTML, CSS static website or a React, Vue, Angular or Next.js Complex app. Zero JS and no plugins installed.",
+    "category": "frontend",
+    "subcategory": "ui-libraries"
+  },
+  {
+    "name": "Float UI",
+    "url": "https://floatui.com/",
+    "description": "Move faster with beautiful, responsive UI components and website templates with modern design, 100% free and open-source.",
```
Comment (CBID2) [inline database/frontend/ui-libraries.json:68]:
```suggestion
    "description": "Move faster with beautiful, responsive UI components and website templates with modern design that are 100% free and open-source.",
```

**Review by CBID2 — CHANGES_REQUESTED** [review]
Hi @JuanPabloDiaz! :) You’re pull request looks great. I found a typo that needs fixing, so you just need to work on that.

File: database/frontend/ui-libraries.json (hunk 1 above)
Comment (JuanPabloDiaz) [reply database/frontend/ui-libraries.json]: I fixed it

**Review by rupali-codes — APPROVED** [review]
lgtm

**Pull request merged**